auto_nag/tests/test_topcrash.py

```diff
@@ -16,4 +22,4 @@ def test_get_blocked_signatures(self):
 
         assert "OOM | small" in signatures
         assert "IPCError-browser | ShutDownKill" in signatures
-        assert "EMPTY: no crashing thread identified; EmptyMinidump" in signatures
+        assert "EMPTY: no frame data available; StreamSizeMismatch" in signatures
```

Comment:
Why did you change the signature here ?

Reply:
It changed from the source, thus I mock the request to avoid the need to update this again in the future.